### DISCUSSION TITLE
I have a coldfusion solution

### DISCUSSION THREAD

**Discussion by anthonyehrhardt** [PlummersSoftwareLLC/Primes#844] [Q&A]
How can I upload the .cfm file?
[primesieve.cfm.txt](https://github.com/PlummersSoftwareLLC/Primes/files/8800994/primesieve.cfm.txt)

**Comment by rbergen** [comment]
You can fork the repository, push your solution to your fork, and open a pull request to add it to this project. All of this is well documented as the "fork and pull" collaborative model [in the GitHub documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models).
